sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -285,6 +285,7 @@ public void testReadingFailsTableDoesNotExist() throws Exception {
 
   /** Tests that when reading from an empty table, the read succeeds. */
   @Test
+  @Category(NeedsRunner.class)
```

Comment:
Because what I understood of annotating this tests is to 'tag' them as tests that use a runner to be executed. In addition the nodeEnforcement changes that stas did get enforced e.g. for TestPipelines with transforms that don't call run().